From b296d4144cc1ec88a577d292f36eccca4a34acb4 Mon Sep 17 00:00:00 2001
From: skyel <razvan.ghitulete@gmail.com>
Date: Tue, 17 Dec 2013 14:35:04 +0100
Subject: [PATCH 05/18] llvmlinux-lto: hack remove LTO_SLIM

This should be done properly, but at the moment there is no way to
compile LTO without SLIM and using SLIM fails our build process

Signed-off-by: skyel <razvan.ghitulete@gmail.com>
---
 scripts/Makefile.lto | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/scripts/Makefile.lto b/scripts/Makefile.lto
index d89ffe9..c51857b 100644
--- a/scripts/Makefile.lto
+++ b/scripts/Makefile.lto
@@ -34,10 +34,6 @@ ifeq ($(call ld-ifversion,-ge,22710001,y),y)
 	LTO_FINAL_CFLAGS := -flto=$(shell getconf _NPROCESSORS_ONLN)
 	#LTO_FINAL_CFLAGS := -flto=jobserver
 
-ifdef CONFIG_LTO_SLIM
-	# requires plugin ar passed and very recent HJ binutils
-        LTO_CFLAGS += -fno-fat-lto-objects
-endif
 # Used to disable LTO for specific files (e.g. vdso)
 	DISABLE_LTO := -fno-lto
 
-- 
1.8.3.1

